ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -65,6 +65,9 @@ public class GrpcLogAppender extends LogAppenderBase {
   private final TimeoutScheduler scheduler = TimeoutScheduler.getInstance();
 
   private volatile StreamObserver<AppendEntriesRequestProto> appendLogRequestObserver;
+  private volatile StreamObserver<AppendEntriesRequestProto> heartbeatRequestObserver;
+  private final boolean useSeparateHBChannel;
+  private final AppendLogResponseHandler appendLogResponseHandler;
```

Comment:
With the new StreamObservers class, this is not needed.

Reply:
Right.